pkg/virt-handler/vm.go

```diff
@@ -1263,6 +1263,36 @@ func (d *VirtualMachineController) updateSELinuxContext(vmi *v1.VirtualMachineIn
 	return nil
 }
 
+func (d *VirtualMachineController) updateVMIStatusFromDomain(vmi *v1.VirtualMachineInstance, domain *api.Domain) error {
+	d.updateIsoSizeStatus(vmi)
+	err := d.updateSELinuxContext(vmi)
+	if err != nil {
+		log.Log.Reason(err).Errorf("couldn't find the SELinux context for %s", vmi.Name)
+	}
+	d.setMigrationProgressStatus(vmi, domain)
+	d.updateGuestInfoFromDomain(vmi, domain)
+	d.updateVolumeStatusesFromDomain(vmi, domain)
+	d.updateFSFreezeStatus(vmi, domain)
+	d.updateMachineType(vmi, domain)
+	err = d.netStat.UpdateStatus(vmi, domain)
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
You can directly return `err`
```suggestion
        return err
```